controllers/sr_controller/sr_controller.py

```diff
@@ -83,13 +96,18 @@ def get_robot_file(zone_id: int, mode: str) -> Path:
     if fallback_robot_file.exists():
         return fallback_robot_file
 
-    print("No robot controller found for zone {}, copying example to {}.".format(
+    print("No robot controller found for zone {} {}, copying example to {}.".format(
         zone_id,
-        fallback_robot_file,
+        robot_type.value,
+        robot_file,
     ))
-    copyfile(str(EXAMPLE_CONTROLLER_FILE), str(fallback_robot_file))
 
-    return fallback_robot_file
+    (controller_utils.ARENA_ROOT / 'zone-0').mkdir(exist_ok=True)
```

Comment:
Meta: What's the thought behind making these directories initially?

Reply:
Since we've moved to populating `zone-0` by default we need to create that directory. The thinking behind creating `zone-1` as well was to make it obvious where to put code for the other zone.